src/transform-sdk/js/CMakePresets.json

```diff
@@ -20,7 +20,7 @@
       "name": "release-type",
       "hidden": true,
       "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Release"
+        "CMAKE_BUILD_TYPE": "RelWithDebInfo"
```

Comment:
q: does this mean we're distributing a binary with debug info?

Reply:
It was an attempt at getting better backtraces, I will revert. I have the tests all working except for the schema registries one now...

Reply:
I'm wondering if I should delay this upgrade, but I don't know if I'll ever do it if I do that 😅

Reply:
ah, i see. makes sense. 

>tests all working except for the schema registries one

yeah, that thing has been nothing but grief. It's possible that there's more info in the transform logs I guess. Happy to take a look at it tomorrow or over the weekend

Reply:
Yeah that's a good point, I should modify the integration tests to print that out after if there is a failure